scripts/fork/helpers/get/getMultiTicketAverageTotalSuppliesBetween.ts

```diff
@@ -0,0 +1,27 @@
+import { Contract } from '@ethersproject/contracts';
+import { BigNumber, BigNumberish } from 'ethers';
+
+export async function getMultiTicketAverageTotalSuppliesBetween(
+  tickets: Array<Contract | undefined> | undefined,
+  startTime?: BigNumberish,
+  endTime?: BigNumberish
+): Promise<BigNumber[] | undefined> {
+  if (!tickets || !startTime || !endTime) return undefined;
```

Comment:
If all these params are not optional, it should be stated in the parameters function. Don't really make sense to return `undefined` here.